_extras/checkout.md

```diff
@@ -45,45 +49,33 @@ with our collaborative curriculum development process.
 
-1.  Submitting a change request to fix an existing issue.
+1.  Submitting a change to fix an existing issue.
 2.  Proof-reading a lesson and adding a new issue describing something to be improved.
-3.  Providing substantive feedback on an existing issue or PR.
+3.  Providing substantive feedback on an existing issue or pull request.
+
+Your contribution must be for a lesson in the curriculum of the Carpentry (Software or Data) for which you are 
+qualifying. Instructors seeking to qualify for both Carpentries, must submit a contribution to both a Data Carpentry
+and a Software Carpentry lesson.
+
+Data Carpentry instructors may submit their contributions by [email](mailto:{{ site.email }}) or through GitHub.
+Software Carpentry instructors must submit their contributions through GitHub.
 
 Trainees must do their work on one of the lessons for which we are currently seeking improvements.
-We encourage trainees to submit their changes through GitHub,
-but will accept submissions for #1 and #2 via email from those who are more comfortable working out of public view.
-
-Pick one of the lessons from the list given later in this section
-and submit a *small* change.
-You must submit a change to a Software Carpentry lesson to qualify as a Software Carpentry instructor,
-and a change to a Data Carpentry lesson to qualify as a Data Carpentry instructor.
-Instructors seeking dual certification should submit two changes (one to each Carpentry)
-to ensure they are familiar with the process and the materials.
-
-Your change can be a new exercise or diagram,
-or an improvement to one or more of the lesson's learning objectives.
+
 Please note that:
 
-1.  If you want to teach Software Carpentry,
-    you must submit the change as a pull request on GitHub.
-    If you want to teach Data Carpentry,
-    you may submit the change either as a pull request
-    or by filing an issue in GitHub that describes your proposed change.
-    Either way,
-    send a link to the pull request or issue to [{{site.email}}](mailto:{{site.email}})
-    so that she can shepherd it through the review process.
+1. If you submit your contribution through GitHub, you need to notify us by email by
+    sending a link to the pull request or issue to [{{site.email}}](mailto:{{site.email}})
+    so that we can track your progress through checkout.
 
-2.  Your change doesn't have to be accepted in order for you to become an instructor:
+2.  Your contribution doesn't need to be accepted in order for you to become an instructor:
     the point is to make sure that you understand the process and have met the people who review material.
 
 3.  Be sure to format your submission in the same way as the content that's already there.
-    For example,
-    lessons should include the 'challenge' style,
-    while learning objectives should be in the quoted block at the top of the lesson.
```

Comment:
We have to be specific about what we mean by formatting - some people in the past have thought we were talking about line wrapping or British vs. American spelling, rather than Markdown-isms.